qtbroker.py

```diff
@@ -70,60 +75,39 @@ def fill_widget(widget, value):
     fill_item(widget.invisibleRootItem(), value)
 
 
-def view_header(header):
-    widget = QTreeWidget()
-    widget.setAlternatingRowColors(True)
-    fill_widget(widget, header)
-    widget.show()
-    return widget
-
-
-class HeaderViewerWidget:
-    def __init__(self, dispatcher, db=None):
-        self.db = db
-        self.dispatcher = dispatcher
-        self._tabs = QTabWidget()
-        self._widget = QWidget()
-        self._tree = QTreeWidget()
-        self._tree.setAlternatingRowColors(True)
-        self._figures = {}
-        self._overplot = {}
-        self._header = None  # the current header, used by export functions
+class TableExportWidget:
+    """
+    A Widget with buttons for exporting run data to tabular formats.
 
+    Parameters
+    ----------
+    header : Header
+    db : Broker
+        This argument will be removed once Headers hold a ref to their Brokers.
+    """
+    def __init__(self, header, db):
+        self.widget = QWidget()
+        self._header = header
+        self._db = db
         export_csv_btn = QPushButton('CSV')
         export_csv_btn.clicked.connect(self._export_csv)
         export_xlsx_btn = QPushButton('Excel')
         export_xlsx_btn.clicked.connect(self._export_xlsx)
-        self._export_btns = [export_csv_btn, export_xlsx_btn]
-        # Disable buttons until first data set is loaded.
-        [btn.setEnabled(False) for btn in self._export_btns]
 
-        tree_container = QVBoxLayout()
-        export_container = QHBoxLayout()
-        export_container.addWidget(export_csv_btn)
-        export_container.addWidget(export_xlsx_btn)
         layout = QHBoxLayout()
-        tree_container.addWidget(QLabel("View Header (metadata):"))
-        tree_container.addWidget(self._tree)
-        tree_container.addWidget(QLabel("Export Events (data):"))
-        tree_container.addLayout(export_container)
-        layout.addLayout(tree_container)
-        layout.addWidget(self._tabs)
-        self._widget.setLayout(layout)
+        layout.addWidget(export_csv_btn)
+        layout.addWidget(export_xlsx_btn)
```

Comment:
maybe a combo box? In case we want to output other formats, like h5 from suitcase?